### DISCUSSION TITLE
Home directory not mounted for Active Directory users

### DISCUSSION THREAD

**Discussion by petertuharsky** [ltsp/ltsp#862] [General]
Hallo, friends.

I have formulated this as discussion topic first, but maybe it better suits a definition of issue...

I'm setting up LTSP server for AD authenticated users. Partially inspired by https://github.com/ltsp/ltsp/discussions/516
I'm using Debian 12, Krb5, Samba, Winbind, without SSSD.

The authentication works fine both on server and LTSP client, but on client the (existing) home directory is not mounted from server. I cannot figure out, why.

I haven't anticipated any problem, it would seem as a quite basic functionality.

The getent passwd user and getent shadow user commands give correct output on both client and server. I am also able to mount the user's dir manually from client using # sshfs user@domain@server: /mnt
(it asks for password)

For non-AD user, the home is mounted correctly from server. Just not for AD users.

Am I missing something, or is there a bug?

**Comment by viniciusferrao** [comment]
I was able to mount without issues using `sssd`. But I got an error message about `pamltsp` at login.

Yes there any reason to not use `sssd`? You can join the image/chroot with `adcli` to ease the process.

**Comment by alkisg** [comment]
Converting into a discussion, please ping me if it ends up being a bug in the LTSP code.
